Mathlib/Logic/Equiv/Defs.lean

```diff
@@ -487,7 +487,7 @@ def propEquivPUnit {p : Prop} (h : p) : p ≃ PUnit.{0} := @equivPUnit p <| uniq
 /-- `ULift α` is equivalent to `α`. -/
 @[simps (config := { fullyApplied := false }) apply]
 protected def ulift {α : Type v} : ULift.{u} α ≃ α :=
-  ⟨ULift.down, ULift.up, ULift.up_down, fun _ => rfl⟩
+  ⟨ULift.down, ULift.up, ULift.up_down, ULift.down_up.{v, u}⟩
```

Comment:
```suggestion
  ⟨ULift.down, ULift.up, ULift.up_down, ULift.down_up⟩
```
for consistency maybe?

Reply:
That change leads to this error:
```
invalid occurrence of universe level 'u_1' at 'Equiv.ulift', it does not occur at the declaration type, nor it is explicit universe level provided by the user, occurring at expression
  ULift.down_up.{v, u_1}
at declaration body
  fun {α : Type v} ↦
    { toFun := ULift.down, invFun := ULift.up, left_inv := (_ : ∀ (b : ULift α), { down := b.down } = b),
      right_inv := (_ : ∀ (a : α), { down := a }.down = a) }
```

Reply:
I don't understand why the explicit universe parameters are required here and I note that even if we supply all the others like this:
```lean
  ⟨ULift.down.{u, v}, ULift.up.{u, v}, ULift.up_down.{v, u}, ULift.down_up.{v, u}⟩
```
then explicit universes are still required on `ULift.down_up`.

So I think we can't do better than what is currently proposed.